pwa/src/templates/templateParts/cronjobsForm/EditCronjobFormTemplate.tsx

```diff
@@ -175,6 +177,20 @@ export const EditCronjobFormTemplate: React.FC<EditCronjobFormTemplateProps> = (
               {errors["throws"] && <ErrorMessage message={errors["throws"].message} />}
             </FormField>
 
+            <FormField>
+              <FormFieldInput className={styles.flex}>
```

Comment:
Class names should be descriptive as to what the objects _are_. Not as to how they look.

E.g.: instead of `.flex`, we could use `.runTagContainer`.

Above being said, I think we can remove the class name _and_ the css altogether, it doesn't seem to do anything.